sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java

```diff
@@ -70,23 +79,30 @@
   private final SettingsService settingsService;
   private final RootsService rootsService;
   private final WebhooksService webhooksService;
-  private final ProjectAnalysisService projectAnalysisService;
+  private final ProjectAnalysisService projectAnalysisOld;
+  private final ProjectAnalysesService projectAnalyses;
   private final NotificationsService notificationsService;
   private final ProjectBranchesService projectBranchesService;
 
   DefaultWsClient(WsConnector wsConnector) {
     this.wsConnector = wsConnector;
-    this.organizations = new OrganizationService(wsConnector);
-    this.permissionsService = new PermissionsService(wsConnector);
-    this.componentsService = new ComponentsService(wsConnector);
+    this.organizationsOld = new OrganizationService(wsConnector);
+    this.organizations = new OrganizationsService(wsConnector);
+    this.permissionsOld = new org.sonarqube.ws.client.permission.PermissionsService(wsConnector);
+    this.permissions = new PermissionsService(wsConnector);
+    this.componentsOld = new org.sonarqube.ws.client.component.ComponentsService(wsConnector);
+    this.components = new ComponentsService(wsConnector);
     this.favoritesService = new FavoritesService(wsConnector);
-    this.qualityProfilesService = new QualityProfilesService(wsConnector);
-    this.issuesService = new IssuesService(wsConnector);
+    this.qualityProfilesOld = new QualityProfilesService(wsConnector);
+    this.qualityprofiles = new QualityprofilesService(wsConnector);
+    this.issuesOld = new org.sonarqube.ws.client.issue.IssuesService(wsConnector);
+    this.issues = new IssuesService(wsConnector);
     this.usersService = new UsersService(wsConnector);
     this.userGroupsService = new UserGroupsService(wsConnector);
     this.userTokensService = new UserTokensService(wsConnector);
     this.qualityGatesService = new QualitygatesService(wsConnector);
-    this.measuresService = new MeasuresService(wsConnector);
+    this.measuresOld = new org.sonarqube.ws.client.measure.MeasuresService(wsConnector);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "MeasuresService"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)